pkg/controller/main-controller.go

```diff
@@ -91,25 +91,26 @@ const (
 
 // Standard Status messages for Tenant
 const (
-	StatusInitialized                = "Initialized"
-	StatusProvisioningCIService      = "Provisioning MinIO Cluster IP Service"
-	StatusProvisioningHLService      = "Provisioning MinIO Headless Service"
-	StatusProvisioningStatefulSet    = "Provisioning MinIO Statefulset"
-	StatusProvisioningConsoleService = "Provisioning Console Service"
-	StatusProvisioningKESStatefulSet = "Provisioning KES StatefulSet"
-	StatusProvisioningInitialUsers   = "Provisioning initial users"
-	StatusProvisioningDefaultBuckets = "Provisioning default buckets"
-	StatusWaitingMinIOCert           = "Waiting for MinIO TLS Certificate"
-	StatusWaitingMinIOClientCert     = "Waiting for MinIO TLS Client Certificate"
-	StatusWaitingKESCert             = "Waiting for KES TLS Certificate"
-	StatusUpdatingMinIOVersion       = "Updating MinIO Version"
-	StatusUpdatingKES                = "Updating KES"
-	StatusNotOwned                   = "Statefulset not controlled by operator"
-	StatusFailedAlreadyExists        = "Another MinIO Tenant already exists in the namespace"
-	StatusTenantCredentialsNotSet    = "Tenant credentials are not set properly"
-	StatusInconsistentMinIOVersions  = "Different versions across MinIO Pools"
-	StatusRestartingMinIO            = "Restarting MinIO"
-	StatusDecommissioningNotAllowed  = "Pool Decommissioning Not Allowed"
+	StatusInitialized                   = "Initialized"
+	StatusProvisioningCIService         = "Provisioning MinIO Cluster IP Service"
+	StatusProvisioningHLService         = "Provisioning MinIO Headless Service"
+	StatusProvisioningStatefulSet       = "Provisioning MinIO Statefulset"
+	StatusProvisioningConsoleService    = "Provisioning Console Service"
+	StatusProvisioningKESStatefulSet    = "Provisioning KES StatefulSet"
+	StatusProvisioningInitialUsers      = "Provisioning initial users"
+	StatusProvisioningInitialUsersStuck = "Provisioning initial users probably stuck, check k8s events"
```

Comment:
Message should be more like "waiting for minio to be available", stating that is stuck implies is failing and might not be true